angular-client/src/environment/environment.prod.ts

```diff
@@ -0,0 +1,4 @@
+export const environment = {
+  production: true,
+  url: 'http://192.168.100.176:3000'
```

Comment:
are these enviroment variables? and if so, did u mean to put them in github?

Reply:
shit i forgot to add them to gitignore